src/main/java/io/cdap/plugin/google/common/GoogleDriveBaseConfig.java

```diff
@@ -123,23 +116,18 @@ public String getReferenceName() {
     return referenceName;
   }
 
-  public String getAccessToken() {
-    return accessToken;
-  }
-
   public String getDirectoryIdentifier() {
     return directoryIdentifier;
   }
 
-  public String getClientId() {
-    return clientId;
-  }
-
-  public String getClientSecret() {
-    return clientSecret;
-  }
-
-  public String getRefreshToken() {
-    return refreshToken;
+  public String getAccountFilePath() {
+    if (AUTO_DETECT_VALUE.equals(accountFilePath)) {
+      accountFilePath = System.getenv(GOOGLE_APPLICATION_CREDENTIALS_ENV_PROP);
+    }
+    if (!new File(accountFilePath).exists()) {
+      throw new RuntimeException(String.format("There is no account file from neither \"Account file path\" property " +
```

Comment:
We need a validation for this.

Reply:
It is validated implicitly during GoogleDriveClient creation in validate(FailureCollector collector). Should I add separate validate method anyway?

Reply:
done